cmake/Modules/FindPNG.cmake

```diff
@@ -9,7 +9,7 @@
 #  PNG_LIBRARY, where to find the PNG library.
 # None of the above will be defined unles zlib can be found.
 # PNG depends on Zlib
-INCLUDE(FindZLIB)
+find_package(ZLIB QUIET REQUIRED)
```

Comment:
Caused a compiler warning.
Using "Include" here feels like using `eval("x = x + 1")`. Just a tiny code smell. I didn't find many opinions online but the few I found [tended](https://cgold.readthedocs.io/en/latest/tutorials/cmake-sources/includes.html) to [agree](https://cmake.org/pipermail/cmake/2008-September/024035.html)

The warning given when using `INCLUDE(FindZLIB)`:
```
CMake Warning (dev) at ...FindPackageHandleStandardArgs.cmake:438 (message):
  The package name passed to `find_package_handle_standard_args` (ZLIB) does
  not match the name of the calling package (PNG).  This can lead to problems
  in calling code that expects `find_package` result variables (e.g.,
  `_FOUND`) to follow a certain pattern.
  ```